AltStore/AppDelegate.swift

```diff
@@ -58,6 +58,34 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool
     {
+        // Copy STDOUT and STDERR to the logging console
```

Comment:
we won't need this since I've added os.log and/or swift-log support already and we can use that to do file writes.